Formula/anycloud.rb

```diff
@@ -0,0 +1,33 @@
+class Anycloud < Formula
+    desc "Lambda alternative that works with multiple cloud providers"
+    homepage "https://github.com/alantech/anycloud"
+    url "https://github.com/alantech/anycloud/archive/v0.1.2.tar.gz"
+    sha256 "3ca6c849aace024c75e5429db95b37e1a5592902557439996508b469cc706370"
+    license "Apache-2.0"
+  
+    depends_on "rust" => :build
+  
+    resource "alan" do
+      url "https://github.com/alantech/alan/releases/download/v0.1.31/alan-macos.tar.gz"
```

Comment:
this should be on `latest` per https://github.com/alantech/alan#installation. Also v0.1.31 add io parallelization and breaks anycloud rn so be careful when recompiling `anycloud.ln` with v0.1.31 and running it locally because I lost a few hours just figuring that out :/

Reply:
Hmm as far as I understand packages on brew need to be versioned, also the checksum will not match for a new release, right?

Reply:
ah I missed the checksum

Reply:
can you pin it to `v0.1.30` then?

Reply:
Done